tests/resources/VCH-Util.robot

```diff
@@ -160,8 +160,66 @@ Get Docker Params
     Run Keyword If  ${tls_enabled} == ${true}  Set Environment Variable  COMPOSE_TLS_VERSION  TLSv1_2
     Run Keyword If  ${tls_enabled} == ${true}  Set Environment Variable  COMPOSE-PARAMS  -H ${dockerHost}
 
+Convert List to String
+    [Arguments]  @{list}
+    Should Not Be Empty  ${list}
+    ${list-string}=  Set Variable  ${EMPTY}
+    :FOR  ${item}  IN  @{list}
+    \   ${list-was-empty}=  Set Variable If  '${list-string}' == '${EMPTY}'  ${True}  ${false}
+    \   ${list-string}=  Run Keyword If  ${list-was-empty}  Set Variable  ${item}
+    \   ...  ELSE  Catenate  SEPARATOR=|  ${list-string}  ${item}
+
+    [Return]  ${list-string}
+
+Add VCH to Removal Exception List
+    [Arguments]  ${vch}=${EMPTY}
+    ${exceptions-string}=  Get Environment Variable  vm-exceptions  ${EMPTY}
+    @{exceptions-list}=  Run Keyword If  '${exceptions-string}' == '${EMPTY}'  Create List
+    @{exceptions-list}=  Run Keyword Unless  '${exceptions-string}' == '${EMPTY}'  Split String  ${exceptions-string}  separator=|
+    Append To List  ${exceptions-list}  ${vch}
+    ${list-string}=  Convert List To String  @{exceptions-list}
+    Set Environment Variable  vm-exceptions  ${list-string}
+    Log To Console  Saved '${list-string}' to removal exceptions
+
+Remove VCH from Removal Exception List
+    [Arguments]  ${vch}=${EMPTY}
+    ${exceptions-string}=  Get Environment Variable  vm-exceptions  ${EMPTY}
+    Return From Keyword If  '${exceptions-string}' == '${EMPTY}'  No Exceptions Found 
+    @{exceptions-list}=  Run Keyword Unless  '${exceptions-string}' == '${EMPTY}'  Split String  ${exceptions-string}  separator=|
+    ${idx}=  Get Index From List  ${exceptions-list}  ${vch}
+    Remove From List  ${exceptions-list}  ${idx}
+    ${len}=  Get Length  ${exceptions-list}
+    ${list-string}=  Run Keyword If  ${len} != 0  Convert List To String  @{exceptions-list}
+    ...  ELSE  Set Variable  ${EMPTY}
+    Set Environment Variable  vm-exceptions  ${list-string}
+    Log To Console  Saved '${list-string}' to removal exceptions
+
+Check If VCH is In Exception
```

Comment:
Minor: inconsistent casing - `Check If VCH is In Exception`. We could say `Is In` or `Check if VCH is in Exception`.